care/facility/migrations/0420_auto_20240312_2318.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.10 on 2024-03-12 17:48
+
+from math import ceil
+
+from django.db import migrations
+
+
+def double_pain_scale(apps, schema_editor):
+    DailyRound = apps.get_model("facility", "DailyRound")
+    records_to_update = []
+    for daily_round in DailyRound.objects.filter(pain_scale_enhanced__isnull=False):
+        scale = daily_round.pain_scale_enhanced.get("scale", None)
+        if scale:
+            daily_round.pain_scale_enhanced["scale"] *= 2
+
+        records_to_update.append(daily_round)
```

Comment:
We can skip doing bulk update on records that does not have scale right?
```suggestion
        if daily_round.pain_scale_enhanced.get("scale"):
            daily_round.pain_scale_enhanced["scale"] *= 2
            records_to_update.append(daily_round)
```